Intersect.Server.Core/Entities/Events/EventPageInstance.cs

```diff
@@ -892,5 +894,10 @@ public bool ShouldDespawn(MapController map)
 
         return false;
     }
+    
+    protected override EntityItemSource GetItemSource()
+    {
+        return null;
+    }
```

Comment:
Same as in `Projectile`, the method isn't nullable so it shouldn't return `null`, this should be showing up as a warning

I think for this one the following might be correct?

```csharp
        return new EntityItemSource
        {
            EntityType = GetEntityType(),
            EntityReference = new WeakReference<IEntity>(this),
            Id = this.MyPage.Id
        };
```